Telegram/SourceFiles/window/window_main_menu.cpp

```diff
@@ -40,6 +40,10 @@ Copyright (c) 2014-2017 John Preston, https://desktop.telegram.org
 
 namespace Window {
 
+	// TODO: move out to lang file or CMake with metainfo.
```

Comment:
I suggest to extract the variables into the CMake config right now, before merge.